hs_explore/management/commands/lda_recommend_resources.py

```diff
@@ -0,0 +1,342 @@
+from collections import defaultdict
```

Comment:
There has been considerable discussion on whether this should be a Jenkins command (like this) or something else. My feeling is that this will be OK in the long run. But we need to eliminate other management commands that we don't need for production.